src/freenet/node/simulator/RealNodeRequestInsertTest.java

```diff
@@ -172,8 +176,8 @@ int insertRequestTest() throws CHKEncodeException, InvalidCompressionCodecExcept
         Node randomNode = nodes[node1];
         //Logger.error(RealNodeRequestInsertTest.class,"Inserting: \""+dataString+"\" to "+node1);
         
-        //boolean isSSK = requestNumber % 2 == 1;
-        boolean isSSK = true;
+        boolean isSSK = requestNumber % 2 == 1;
+        //boolean isSSK = true;
```

Comment:
The more general test is preferable here IMHO. But it's also useful to be able to toggle it, and this is testing code anyway, so having the commented out version is acceptable?